### PR TITLE
pkill: add --oldest example and link

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pkill.md [hunk 1]
```diff
@@ -2,15 +2,20 @@
 
 > Signal process by name.
 > Mostly used for stopping processes.
+> More information: <https://www.man7.org/linux/man-pages/man1/pkill.1.html>.
 
 - Kill all processes which match:
 
 `pkill -9 {{process_name}}`
```
Comment (bl-ue) [inline pages/common/pkill.md:9]:
```suggestion
`pkill -9 "{{process_name}}"`
```

File: pages/common/pkill.md [hunk 2]
```diff
@@ -2,15 +2,20 @@
...
 
 - Kill all processes which match their full command instead of just the process name:
 
-`pkill -9 -f "{{command_name}}"`
+`pkill -9 --full "{{command_name}}"`
 
 - Send SIGUSR1 signal to processes which match:
 
 `pkill -USR1 {{process_name}}`
```
Comment (bl-ue) [inline pages/common/pkill.md:17]:
```suggestion
`pkill -USR1 "{{process_name}}"`
```

**Review by bl-ue — COMMENTED** [review]
Two examples have quotes and two don't. 🤔

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel! I didn't know about `--oldest` on `pkill` - awesome!

**Pull request merged**